exercises/11/build_and_run/README.md

```diff
@@ -32,10 +39,19 @@ brew install cmake ninja
 
 ## Configure and build project
 
-### All Platforms
+### All Platforms, using VS Code
 
 ```bash
 git clone https://github.com/mod-cpp/pacman.git
 cd pacman
 code .
 ```
+
+### Linux, BSD, or Mac, not using VS Code
```

Comment:
For now we want to maintain the same developer experience cross platform - and in that, we we are guiding the students to use an integrated IDE - primarily VSCode